R/validate_inputs.R

```diff
@@ -68,3 +68,31 @@ assert_area_id_exists <- function(json) {
   }
   invisible(TRUE)
 }
+
+#' Validate population file.
+#'
+#' Check that population file can be read.
+#'
+#' @param shape Path to input population file.
+#'
+#' @return An error if invalid.
+#' @keywords internal
+do_validate_population <- function(population) {
+  population <- read_csv(population, header = TRUE)
+  ## Perhaps this kind of assertion should just be checked by json schema?
+  assert_column_names(
```

Comment:
One question here about the validation - this kind of thing can be checked by json schema for the response data. But there is a question of where is it best to put those checks. Feels to me like checking this explicitly here is the way to go because it is the kind of error with message that we would want to return to the front end instead of having them try to interpret json schema failure messages. 

Actually the more I think about it I think this is the right way to go. We should never rely on failing a schema to validate our data as it will be a real pain to parse into useful messages I think. Interested to hear what you think and then I will remove line 82.